src/Command/App/NewCommand.php

```diff
@@ -52,17 +53,26 @@ protected function execute(InputInterface $input, OutputInterface $output): int
       $dir = '/home/ide/project';
     }
     else {
-      $dir = Path::makeAbsolute('drupal', getcwd());
+      $dir = Path::makeAbsolute($project, getcwd());
```

Comment:
It looks like this line is no longer covered by tests, can you please make sure that coverage doesn't decrease with this PR?

Reply:
For now added a new assertion to check for absolute path for directory here https://github.com/acquia/cli/pull/1084/files#diff-50ab4f1a71885b242b21e893d786e6c7e1305ac3640ba9f33f94fa14c09289ebR130